modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/DataLayoutResourceTest.java

```diff
@@ -41,6 +48,49 @@ public void setUp() throws Exception {
 			irrelevantGroup);
 	}
 
+	@Override
+	@Test
+	public void testGetDataDefinitionDataLayoutsPage() throws Exception {
+		super.testGetDataDefinitionDataLayoutsPage();
```

Comment:
@brianchandotcom @shuyangzhou problem is this line. `super.testGetDataDefinitionDataLayoutsPage()` generates random data layouts and don't delete them. Some lines bellow (line 63), it tries to search for "!@#l", which can return some of the random data layouts just created. That's why it's intermittent, because object fields are random.

@nhpatt, can we modify the generator to delete the entities it creates? I guess we do that during tearDown but it's not enough since we call that test directly. 

cc @jeyvison @gabrielwas

Reply:
Bruno and I discussed several options in slack, I can do the change if the solutions don't fit their needs for the tests :)